src/platform/site-wide/announcements/config/index.js

```diff
@@ -65,6 +66,11 @@ const config = {
       paths: /^\/$/,
       component: WelcomeToNewVAModal,
     },
+    {
+      name: 'single-sign-on-intro',
+      paths: /^\/$/,
```

Comment:
while a match for any page, note that the component has logic to only render for a logged in user with an SSO session